templates/shared/pack.js

```diff
@@ -0,0 +1,12 @@
+const { readFileSync } = require('fs');
+const { parse, resolve } = require('path');
+const AdmZip = require('adm-zip');
+
+const { base } = parse(__dirname);
+const { version } = JSON.parse(
+  readFileSync(resolve(__dirname, 'build', 'manifest.json'), 'utf8')
+);
+
+const zip = new AdmZip();
+zip.addLocalFolder('build');
+zip.writeZip(`${base}-v${version}.zip`);
```

Comment:
It would be good if we can log a success message after zip file is generated.

Something like this:
```js
console.log(`${chalk.green('Success!')} Created a ${base}-v${version}.zip file. You can upload this file to web store.`);
```